app/1.0/blog/2016-08-18-polymer-starter-kit-or-polymer-cli.md

```diff
@@ -0,0 +1,108 @@
+---
+title:  "Should I Use Polymer Starter Kit or Polymer CLI"
+---
+
+For many developers new to Polymer, the
+[Polymer Starter Kit](https://github.com/polymerelements/polymer-starter-kit) 
+project is a useful seed to get the ball rolling on their own app. With the 
+release of [Polymer CLI](https://www.polymer-project.org/1.0/docs/tools/polymer-cli), 
+which contains its own starter projects, it became unclear to developers which 
+resource they should choose when creating a new project.
+
+To make it as easy as possible to get started building Polymer projects we’ve 
+turned Polymer Starter Kit into a template that can be used with Polymer CLI!
+
+![An example page showing Polymer Starter Kit in the browser](images/starter-kit-preview.jpg)
+
+Update your version of polymer-cli:
```

Comment:
This should be either "Polymer CLI" or `polymer-cli` in code font. I'd vote for the former.